napari/components/scale_bar.py

```diff
@@ -23,9 +24,15 @@ class ScaleBar(EventedModel):
     background_color : np.ndarray
         Background color of canvas. If scale bar is not colored
         then it has the color opposite of this color.
+    font_size : float
+        The font size (in points) of the text.
+    unit : str
+        Unit to be used by the scale bar.
     """
 
     visible: bool = False
     colored: bool = False
     ticks: bool = True
     position: Position = Position.BOTTOM_RIGHT
+    font_size: float = 10
+    unit: str = ""
```

Comment:
Maybe use `NoUnit` here as the default value?

Reply:
Does pint have some concept of no unit? We could also use `None` here too, i.e. `unit: Optional[str] = None`? I might have a slight preference for that over the `NoUnit` string. Curious what others think too?

Reply:
>Does pint have some concept of no unit? We could also use None here too, i.e. unit: Optional[str] = None? I might have a slight preference for that over the NoUnit string. Curious what others think too?

Yes, `None` feels more natural to work with.